.github/workflows/ci.yml

```diff
@@ -159,3 +159,34 @@ jobs:
         with:
           directory: coverage
           #files: lcov.info
+
+  doc:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        rust: [nightly]
+
+    name: Build-test documentation using Rust ${{ matrix.rust }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v2
+
+      - name: Install rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ matrix.rust }}
+          override: true
```

Comment:
I think you can add `components: rust-docs` here to make this a little faster.

Reply:
Isn't that additional, on top of what is already installed by `profile: minimal`?

(I'm surprised that this works, since `rust-docs` is only included in https://rust-lang.github.io/rustup/concepts/profiles.html - perhaps it's pulling it from GH's runner/)

Reply:
EDIT: Perhaps it doesn't work. The CI in this PR seems to run the workflow from `master`, so _excluding_ my job (something to do with possibly exposing secrets without maintainer approval, I suppose), and I wouldn't have caught this as the `on:` branch filter doesn't allow my branchname.